test/journeys/lib/test-helpers/space-widget/featureFlags.js

```diff
@@ -0,0 +1,48 @@
+import {assert} from 'chai';
+
+import {elements as mainElements} from './main';
+import {elements as rosterElements} from './roster';
+import {elements as meetElements} from './meet';
+
+/**
+ *
+ * Feature Flag Tests Journeys
+ *
+ * @export
+ * @param {any} browserLocal
+ * @param {any} browserRemote
+ * @returns {null}
+ */
+export default function featureFlagTests(browserLocal, browserRemote) {
+  describe(`Roster Feature Flag`, () => {
+    it(`has a roster for user with feature flag`, () => {
+      browserLocal.click(mainElements.menuButton);
+      browserLocal.waitForVisible(mainElements.activityMenu);
+      assert.isTrue(browserLocal.element(mainElements.controlsContainer).element(rosterElements.peopleButton).isVisible());
+      browserLocal.click(mainElements.exitButton);
+    });
+
+    it(`does not have a roster for user without flag`, () => {
+      browserRemote.click(mainElements.menuButton);
+      browserRemote.waitForVisible(mainElements.activityMenu);
+      assert.isFalse(browserRemote.element(mainElements.controlsContainer).element(rosterElements.peopleButton).isVisible());
+      browserRemote.click(mainElements.exitButton);
+    });
+  });
+
+  describe(`Group Calling Feature Flag`, () => {
+    it(`has a call option for user with feature flag`, () => {
+      browserLocal.click(mainElements.menuButton);
+      browserLocal.waitForVisible(mainElements.activityMenu);
+      assert.isTrue(browserLocal.element(mainElements.controlsContainer).element(meetElements.callButton).isVisible());
+      browserLocal.click(mainElements.exitButton);
+    });
+
+    it(`does not have a call option for user without flag`, () => {
+      browserRemote.click(mainElements.menuButton);
+      browserRemote.waitForVisible(mainElements.activityMenu);
+      assert.isFalse(browserRemote.element(mainElements.controlsContainer).element(meetElements.callButton).isVisible());
+      browserRemote.click(mainElements.exitButton);
+    });
+  });
```

Comment:
May want to rename `browserRemote` and `browserLocal` to something more obvious to say that one has the feature flag enabled and one does not. Also does it make more sense to put the feature flag tests with the feature tests instead of here?

Reply:
* Good call on the renaming. 
* My thought on keeping the "has feature with flag, doesn't have it without it" tests were that we can keep track of all the feature flags in one test suite, while testing actual functionality in the others.